docs/guides/ORCHESTRATION_CHAT_COMPLETION.md

```diff
@@ -300,6 +302,73 @@ Note, that only user and system messages are supported for multiple text inputs.
 Please find [an example in our Spring Boot application](../../sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/services/OrchestrationService.java).
 
 
+## Set a Response Format
+
+It is possible to set the response format for the chat completion. Available options are using `JSON_OBJECT` and `JSON_SCHEMA`.
+
+### JSON_OBJECT
+
+If you want to force the response to be a JSON object, you can set the response format as follows:
```

Comment:
This sentence doesn't explain anything, what is a response object, how is it different to a json schema?